src/test/kotlin/org/rust/RsTestBase.kt

```diff
@@ -212,8 +213,8 @@ abstract class RsTestBase : BasePlatformTestCase(), RsTestCase {
                 error("Can't mix `BothEditions` and `MockEdition` annotations")
             }
             // These functions exist to simplify stacktrace analyzing
-            runTestEdition2015(testRunnable)
             runTestEdition2018(testRunnable)
+            runTestEdition2015(testRunnable)
```

Comment:
Do you intentionally change order here?

Reply:
Yes, otherwise tests didn't pass because of `setEdition` bug. I fixed the bug and reverted the order